src/Error.cpp

```diff
@@ -45,5 +45,12 @@ namespace error
               concatVector(errorLocation_in) + "': " + std::move(what))
         , errorLocation(std::move(errorLocation_in))
     {}
+
+    Internal::Internal(std::string const &what)
+        : Error(
+              "Internal error: " + what +
+              "\nThis is a bug. Please report at ' "
+              "https://github.com/openPMD/openPMD-api/issues'.")
```

Comment:
Hard to copy, did read `' url'.`
```suggestion
              "\nThis is a bug. Please report at "
              "https://github.com/openPMD/openPMD-api/issues")
```